CHANGELOG.txt

```diff
@@ -226,7 +226,7 @@ See https://matthewmiller.dev/blog/introducing-worldedit-7/ for a friendlier exp
 - Added //drawsel, a Server-Side WorldEditCUI for small cuboid selections (< 32x32x32)
 - Rewrote the block parser, now properly handles Minecraft 1.13 names (See this blog post for more information)
 - Switched to the Sponge Schematic format, as the MCEdit Schematic Format does not work in 1.13
-    - Old schematics will be loaded and converted as best as possible
+    - Old schematics will be loaded and converted as good as possible
```

Comment:
I somewhat prefer the original - but neither are actually correct.

Probably best for this to be `well`

Reply:
"well" does not fit either IMO, my proposal:
```suggestion
    - Old schematics will be loaded and converted, though the conversion may not be 100% accurate
```